ui/src/components/inputs/LowCodeEditor.vue

```diff
@@ -278,22 +280,17 @@
     };
 
     const onCreateNewTask = (details) => {
```

Comment:
This looks like an event with a payload.
Is it too deep to trigger this event? Should we inject a `create` & `edit` function instead?

Reply:
You mean to ditch the `topologyClick` injection altogether and inject those two function instead?

Reply:
Exactly! 
Since the intent is an action, we should not create a value that will mutate and be stored in the state. 

This is how I think of it:

Action => Function/event/mutation
Stateful properties => Variable, State engines (vuex, pinia)

Reply:
TBH, I still am not completely sure what would you ant me to do here... We are now watching that `topologyClick` in three places, in EditorWrapper, where the highlighting is being done if only `YAML` editor is shown, in the `NoCode` where we react to click if `NoCode` editor is visible and in `useCodeTopology` where we handle situation when none of the editors is visible.

Reply:
It is a question of taste. 
I find watchers and synced variables hard to work with/optimize compared to computed/events.

Let's go with what you have. I am sure I am freaking out for nothing.
If we do feel the pain later, we can fix it then.